build/root/WORKSPACE

```diff
@@ -12,7 +12,7 @@ http_archive(
     urls = ["https://github.com/kubernetes/repo-infra/archive/4291717be0a0f9e789c800c295f2a4067de15bef.tar.gz"],
 )
 
-ETCD_VERSION = "3.0.17"
+ETCD_VERSION = "3.1.10"
 
 new_http_archive(
```

Comment:
The "sha256" value in this rule is going to change. I can't comment on the line but it's line 20. The new hash is "2d335f298619c6fb02b1124773a56966e448ad9952b26fea52909da4fe80d2be" according to the build failure.